bedrock/firefox/views.py

```diff
@@ -579,19 +582,19 @@ def new(request):
                 template = 'firefox/new/sem/unsupported-browser/scene1.html'
             elif experience == 'batmfree':
                 template = 'firefox/new/batm/free.html'
-            elif experience == 'batmprivate':
-                if variant == 'a':
-                    template = 'firefox/new/batm/machine-a.html'
-                elif variant == 'b':
-                    template = 'firefox/new/batm/machine-b.html'
-                else:
-                    template = 'firefox/new/batm/private.html'
             elif experience == 'batmnimble':
                 template = 'firefox/new/batm/nimble.html'
             elif experience == 'batmresist':
                 template = 'firefox/new/batm/resist.html'
+            elif experience == 'batmprivate':
+                if variant == 'b':
+                    template = 'firefox/new/batm/machine.html'
+                else:
+                    template = 'firefox/new/batm/private.html'
             else:
                 template = 'firefox/new/onboarding/scene1.html'
+        elif lang_file_is_active('firefox/new/batm', locale) and experience == 'batmprivate':
+            template = 'firefox/new/batm/private.html'
```

Comment:
Looking at this again, do we need to do the same `if variant == 'b'` check in this instance?